frontend/src/__tests__/cypress/cypress/tests/mocked/pipelines/pipelines.cy.ts

```diff
@@ -597,6 +598,24 @@ describe('Pipelines', () => {
     pipelineImportModal.findSubmitButton().should('be.enabled');
   });
 
+  it('imports fails with Argo workflow', () => {
```

Comment:
you are missing test coverage on other areas that were changed

Reply:
added few more test cases, let me if any thing i've not covered